src/lib/annotations/__tests__/Annotator-test.js

```diff
@@ -560,7 +536,7 @@ describe('lib/annotations/Annotator', () => {
                     'click',
                     sinon.match.func
                 );
-                expect(annotator.annotationModeHandlers.length).equals(4);
+                expect(annotator.annotationModeHandlers.length).equals(6);
```

Comment:
Add tests for when only some of the buttons exist i.e postButtonEl, undoButton...etc